modules/docker/docker_compose.go

```diff
@@ -16,15 +16,25 @@ type Options struct {
 
 // RunDockerCompose runs docker-compose with the given arguments and options and return stdout/stderr.
 func RunDockerCompose(t testing.TestingT, options *Options, args ...string) string {
-	out, err := RunDockerComposeE(t, options, args...)
+	out, err := runDockerComposeE(t, false, options, args...)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return out
 }
 
+// RunDockerComposeAndGetStdout runs docker-compose with the given arguments and options and returns only stdout.
+func RunDockerComposeAndGetStdOut(t testing.TestingT, options *Options, args ...string) string {
+	out, _ := runDockerComposeE(t, true, options, args...)
```

Comment:
Don't ignore errors, as it will making debugging very hard. 

```suggestion
	out, err := runDockerComposeE(t, true, options, args...)
	require.NoError(t, err)
```